addons/survey/tests/test_survey_security.py

```diff
@@ -140,25 +140,34 @@ def test_access_survey_survey_manager(self):
     @mute_logger('odoo.addons.base.models.ir_model')
     @users('survey_user')
     def test_access_survey_survey_user(self):
-        # Create: own only
-        survey = self.env['survey.survey'].create({'title': 'Test Survey 2'})
-        self.env['survey.question'].create({'title': 'My Page', 'sequence': 0, 'is_page': True, 'question_type': False, 'survey_id': survey.id})
-        self.env['survey.question'].create({'title': 'My Question', 'sequence': 1, 'survey_id': survey.id})
-
-        # Read: all
+        # Restrict common survey to survey_manager
+        restricted_to_other_survey = self.survey
+        restricted_to_other_survey.write({'restrict_user_ids': [[4, restricted_to_other_survey.user_id.id]]})
```

Comment:
Would probably be more explicit (otherwise you have to regrep everything to understand what are the values)

And/Or assert user_id explicitly to be sure (and make test easier to read)